fineract-provider/src/main/java/org/apache/fineract/adhocquery/service/AdHocScheduledJobRunnerServiceImpl.java

```diff
@@ -87,9 +88,8 @@ public void generateClientSchedule() {
                                 run = Days.daysBetween(start, end).getDays() >= adhoc.getReportRunEvery();
                             break;
                             default:
-                                throw new IllegalStateException();
+                                throw new IllegalArgumentException();
```

Comment:
If you have suppressed this check..Do we still need this default?

Reply:
Was my mistake wasn't not suppose to change the exception will revert it. Yeah we need it because Error prone thinks it is not necessary since all cases are specified.